roles/wildfly_systemd/tasks/service.yml

```diff
@@ -12,3 +12,5 @@
     enabled: "{{ wildfly_systemd.enabled | default(omit) }}"
     daemon_reload: true
   become: yes
+  tags:
+    - molecule-idempotence-notest
```

Comment:
Do we want to do that?  @guidograzioli correct me if I'm wrong, but this will prevent molecule to fail the idempotency run, right? Should we not thrive to make this tasks idempotent?

Reply:
definitely not wanted, in particular on the systemd role!